connector/kafka2ftp/app/KafkaFTPConsumers.js

```diff
@@ -13,21 +11,21 @@ class KafkaFTPConsumers {
    *
    */
   constructor() {
-    logger.debug('constructor: Instance Kafka', TAG);
-    this.consumer = new ConsumerBackPressure(consumerConfig);
+    logger.debug('constructor: Instance Kafka');
+    this.consumer = new Consumer({ 'kafka.consumer': consumerConfig });
```

Comment:
Shouldn't we pass the topic settings too (second parameter of the constructor)?

Reply:
The configurations that existed were only of the consumer type and the ConfigManager was not used, so currently no topic configuration is passed. Do you think I should put topic settings?

Reply:
Let's keep this way.